### PR TITLE
Update cron-jobs.md

### PR DESCRIPTION
**Pull request by adi-wan-askui**
fix typo

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ That is, the CronJob does _not_ update existing Jobs, even if those remain runni
 A CronJob creates a Job object approximately once per execution time of its schedule.
 The scheduling is approximate because there
 are certain circumstances where two Jobs might be created, or no Job might be created.
-Kubernetes tries to avoid those situations, but do not completely prevent them. Therefore,
+Kubernetes tries to avoid those situations, but does not completely prevent them. Therefore,
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/workloads/controllers/cron-jobs.md:205]: The small trivial change looks good to me.

**Comment by dipesh-rawat** [comment]
Looks like the Netlify build are failing with error `Build exceeded maximum allowed runtime` as mentioned in https://github.com/kubernetes/website/issues/35780

**Comment by dipesh-rawat** [comment]
@adi-wan-askui Thanks for the fix. Could you please try changing the commit message for the last commit and then force push, that will trigger a new preview build.

**Comment by tengqm** [comment]
Site build looks okay, the timeout is a transient issue.
/lgtm
/approve

**Pull request merged**